openedx/features/redhouse_panel/utils.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+from django.conf import settings
+
+from openedx.features.edly.utils import (
+    get_edx_org_from_cookie,
+    update_course_creator_status,
+)
+
+from student.roles import (
+    CourseInstructorRole,
+    CourseStaffRole,
+    GlobalCourseCreatorRole,
+    GlobalStaff,
+    UserBasedRole,
+)
+
+
+def set_global_course_creator_status(request, user, set_global_creator):
+    """
+    Updates global course creator status of a user.
+    """
+    from course_creators.models import CourseCreator
+    from course_creators.views import update_course_creator_group
+
+    request_user = request.user
+
+    course_creator, __ = CourseCreator.objects.get_or_create(user=user)
```

Comment:
```suggestion
    course_creator, _ = CourseCreator.objects.get_or_create(user=user)
```

Reply:
@danialmalik it's better to use double underscore for unused values in Django as most of the times we make use a single underscore as a translation function e.g

```
from django.utils.translation import ugettext_lazy as _

code = models.CharField(
    help_text=_("The ISO 639-1 language code for this language.")
)
```

Reply:
Done